source/core/src/main/com/csse3200/game/inventory/items/potions/AttackPotion.java

```diff
@@ -0,0 +1,64 @@
+package com.csse3200.game.inventory.items.potions;
+
+import com.csse3200.game.components.CombatStatsComponent;
+import com.csse3200.game.inventory.items.ItemUsageContext;
+import com.csse3200.game.services.GameTime;
+
+import java.util.Timer;
+import java.util.TimerTask;
+
+public class AttackPotion extends  AbstractPotion{
+    private final static String path = "images/potiontexture/attack.png";
```

Comment:
Abstract away these types of potions which have duplicate code - they all just check game time from when they were used and then have some behaviour for when they are completed.